dataproc/create_hail_cluster.ipynb

```diff
@@ -0,0 +1,865 @@
+    "    --master-machine-type n1-standard-4 \\\n",
+    "    --master-boot-disk-size 100 \\\n",
+    "    --autoscaling-policy two_worker_autoscaling_policy \\\n",
+    "    --num-workers 2 \\\n",
+    "    --worker-machine-type n1-standard-4 \\\n",
+    "    --worker-boot-disk-size 100 \\\n",
+    "    --num-secondary-workers 0 \\\n",
+    "    --secondary-worker-machine-type n1-standard-4 \\\n",
+    "    --secondary-worker-boot-disk-size 100 \\\n",
+    "    --enable-component-gateway \\\n",
+    "    --optional-components JUPYTER \\\n",
+    "    --service-account {SERVICE_ACCOUNT} \\\n",
+    "    --subnet projects/{PROJECT}/regions/us-central1/subnetworks/subnetwork \\\n",
+    "    --tags leonardo \\\n",
+    "    --bucket {STAGING_BUCKET[len('gs://'):]} \\\n",
+    "    --temp-bucket {TEMP_BUCKET[len('gs://'):]} \\\n",
+    "    --max-idle 30m"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b860d05f-0788-4e4e-b402-3dda1abb7c17",
+   "metadata": {},
+   "source": [
+    "The cluster will take a few minutes to start up."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "73de7bc8-c2ac-46bd-9587-5f88cca60265",
+   "metadata": {},
+   "source": [
+    "## Access JupterLab and the debugging consoles\n",
+    "\n",
+    "You can use the URL printed by the next cell to access JupyterLab running on the cluster. See also the URLs to the debuging consoles such as the Spark Console.\n",
+    "\n",
+    "Or if you would like to use the Cloud Console to obtain these URLs:\n",
+    "* Go to the Cloud Console -> Dataproc -> Clusters\n",
+    "* Select the cluster on which you want to run the notebook\n",
+    "* Click on tab 'WEB INTERFACES'\n",
+    "* Click on 'JupyterLab'\n",
+    "\n",
+    "Lastly, CPU utilization, memory utlization, and other performance metrics for the cluster are available on the Cloud Console. Click on the cluster name to see the plots of these metrics."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c5d3c60c-b616-45e7-b1be-c3cc8625d7fc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!gcloud dataproc clusters describe {HAIL_CLUSTER_NAME} --region=us-central1 --format=\"yaml(config.endpointConfig.httpPorts)\""
```

Comment:
nit: line is a bit too long, consider adding slash-newline so the bash code displays well on GitHub